files/en-us/web/api/webgl_api/webgl_best_practices/index.html

```diff
@@ -625,23 +621,21 @@ <h3 id="texImagetexSubImage_uploads_particularly_with_videos_can_cause_pipeline_
     ...
 </code></pre>
 
-<h2 id="WebGL_2">WebGL 2</h2>
+<h2 id="Use_texStorage_to_create_textures">Use texStorage to create textures</h2>
 
-<h3 id="Use_texStorage_to_create_textures">Use texStorage to create textures</h3>
-
-<p>The <code>texImage*</code> API lets you define each mip level independently and at any size, even the mis-matching mips sizes are not an error until draw time which means there is no way the driver can actually prepare the texture in GPU memory until the first time the texture is drawn.</p>
+<p>The WebGL 2.0 <code>texImage*</code> API lets you define each mip level independently and at any size, even the mis-matching mips sizes are not an error until draw time which means there is no way the driver can actually prepare the texture in GPU memory until the first time the texture is drawn.</p>
 
 <p>Further, some drivers might unconditionally allocate the whole mip-chain (+30% memory!) even if you only want a single level.</p>
 
 <p>So, prefer <code>texStorage</code>+<code>texSubImage</code> for textures in WebGL 2</p>
 
-<h3 id="invalidateFramebuffer">invalidateFramebuffer</h3>
+<h2 id="invalidateFramebuffer">invalidateFramebuffer</h2>
```

Comment:
```suggestion
<h2 id="Use_invalidateFramebuffer">Use invalidateFramebuffer</h2>
```
(or "Consider...")?

(the other H2s are generally in a command-form)

Reply:
Thanks, done in the forthcoming commit.